opentelemetry-api/src/opentelemetry/configuration/__init__.py

```diff
@@ -19,27 +19,27 @@
 Simple configuration manager
 
 This is a configuration manager for OpenTelemetry. It reads configuration
-values from environment variables prefixed with
-``OPENTELEMETRY_PYTHON_`` whose characters are only all caps and underscores.
-The first character after ``OPENTELEMETRY_PYTHON_`` must be an uppercase
-character.
+values from environment variables prefixed with ``OPENTELEMETRY_PYTHON_`` whose
+characters are only alphanumeric characters and unserscores, except for the
+first character after ``OPENTELEMETRY_PYTHON_`` which must not be a number.
```

Comment:
Just curious, why must the first character after `OPENTELEMETRY_PYTHON_` not be a number?

Reply:
Those env variables are mapped to Python attributes, it's not possible to have a Python attribute name starting by a number :)